authorization-dotnet/BouncyCastleCryptography/BouncyCastleCryptographyTests/AsymmetricEncryptionUnitTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using BouncyCastleCryptography.AsymmetricEncryption;
+using Org.BouncyCastle.Crypto;
+
+
+namespace BouncyCastleCryptographyTests
+{
+    [TestClass]
+    public class AsymmetricEncryptionUnitTests
+    {
+        [TestMethod]
+        public void GivenSecretData_WhenRsaEncrypting_ThenDataSignatureIsProduced()
+        {
+            string input = "Hello, Bouncy Castle!";
+
+            AsymmetricCipherKeyPair keyPair = RsaEncryptor.GenerateRsaKeyPair();
+
+            byte[] encryptedBytes = RsaEncryptor.RsaEncrypt(input, keyPair.Public);
+
+            string decryptedString = RsaEncryptor.RsaDecrypt(encryptedBytes, keyPair.Private);
+
+            Console.WriteLine("Decrypted data: " + decryptedString);
```

Comment:
Console.WriteLine is not required in unit test cases. Please review it from all the test cases.